Src/Libretro/Glob.c

```diff
@@ -135,7 +135,10 @@ ArchGlob* archGlob(const char* pattern, int flags)
     globHandle = (ArchGlob*)calloc(1, sizeof(ArchGlob));
 
     for (i = 0; i < g.gl_pathc; i++) {
-        char* path = g.gl_pathv[i];
+        //char* path = g.gl_pathv[i];
+        char path[1024];
+        sprintf(path,"%s\0",g.gl_pathv[i]);
```

Comment:
Please don't do this... you're only adding more bugs. If `g.gl_pathv[i]` is longer than 1024, you will overflow the `path` buffer and crash again. And I don't see why you'd want to use a fixed size buffer here anyway.

The real issue needs to be solved here instead of trying to get around something.

Reply:
yeah, i will remove PR
I let you find the bug then...